vendors/st/boards/stm32l475_discovery/aws_tests/application_code/main.c

```diff
@@ -66,6 +66,9 @@ void vApplicationDaemonTaskStartupHook( void );
 extern void SPI_WIFI_ISR(void);
 extern SPI_HandleTypeDef hspi;
 
+/* Setup cellular connection. */
+bool prvSetupCellular( void );
```

Comment:
Is this a private (i.e. file static) function?  If not, it should not be named prv. Also, does the call site need to check return value?

Reply:
Originally, it is designed as a private function. It should only be used in the main.c file.
To reduce the modification in main.c, the implementation of prvSetupCellular is moved to cellular_setup.c.

The return value should be checked. A message will be added if this function returns fail.